src/openforms/registrations/admin.py

```diff
@@ -12,3 +19,22 @@ def formfield_for_dbfield(self, db_field, request, **kwargs):
             return field
 
         return super().formfield_for_dbfield(db_field, request, **kwargs)
+
+
+class TestPluginAdmin(admin.ModelAdmin):
+    # list_display = ('name', 'status')
+    # actions = [test_plugin]
+    change_list_template = 'admin/plugin_tester.html'
+
+    def changelist_view(self, request, extra_context=None):
+        context = extra_context or {}
+        context = {'plugins': [
+            {'name': 'ZGW API', 'test': zwgr.test_config()},
+            {'name': 'StUF-ZDS', 'test': szr.test_config()},
+            {'name': 'Objects API', 'test': oar.test_config()},
+            # {'name': 'Local', 'test': dfr.test_config()},
+        ], 'email': {'name': 'Email', 'test': er.test_config()},}
```

Comment:
This works, but it's nicer to just iterate over all plugins in the `openforms.registratrations.registry.register`. We also don't need to import each plugin in that case.